code/src/pages/dashboard/dashboard-product/components/dashboard-product-container/DashboardProductContainer.tsx

```diff
@@ -0,0 +1,234 @@
+        <AppButton
+          data-cy="new-product-button"
+          variant="contained"
+          to={routes.dashboard.products.update.path(productId)}
+        >
+          <EditIcon />
+          <AppTypography translationKey="product.edit" />
+        </AppButton>
+      </AppBox>
+      <AppBox>
+        <AppBox className="dashboard-product">
+          <AppBox className="dashboard-product__container dashboard-product__image-section">
+            <AppBox className="dashboard-product__header">
+              <AppTypography
+                component="h1"
+                variant="h3"
+                translationKey="dashboardProduct.section.image.title"
+                className="dashboard-product__header-title"
+              />
+            </AppBox>
+            <AppBox className="dashboard-product__body">
+              <AppBox
+                component="img"
+                className="dashboard-product__image"
+                data-testid="product-image"
+                src={product.image}
+                alt={productName}
+              />
+            </AppBox>
+          </AppBox>
+          <AppBox className="dashboard-product__container dashboard-product__additional-info-section">
+            <AppBox className="dashboard-product__header">
+              <AppTypography
+                component="h1"
+                variant="h3"
+                translationKey="dashboardProduct.section.additionalInformation.title"
+                className="dashboard-product__header-title"
+              />
+            </AppBox>
+            <AppBox className="dashboard-product__body">
+              <AppBox className="dashboard-product__additional-info-item">
+                <AppTypography
+                  className="dashboard-product__additional-info-item-label"
+                  translationKey="product.quantity"
+                />
+                <AppTypography>{product.quantity}</AppTypography>
+              </AppBox>
+              <AppBox className="dashboard-product__additional-info-item">
+                <AppTypography
+                  className="dashboard-product__additional-info-item-label"
+                  translationKey="product.price"
+                />
+                <AppTypography>{formatPrice(product.price)}</AppTypography>
+              </AppBox>
+              <AppBox className="dashboard-product__additional-info-item">
+                <AppTypography
+                  className="dashboard-product__additional-info-item-label"
+                  translationKey="product.category"
+                />
+                {displayedCategory}
+              </AppBox>
+            </AppBox>
+          </AppBox>
+          <AppBox className="dashboard-product__main-info-section">
+            <AppBox
+              className="dashboard-product__container dashboard-product__main-info-container"
+              data-testid={`main-info-${product.productTranslations[selectedLocaleIndex].languageCode}`}
+            >
```

Comment:
I suggest moving this complex statement above return

Reply:
Now it is a little bit shorter `productTranslations[selectedLocaleIndex].languageCode`. I think it is not that complex, there is no reason to move it up.